plugins/extractors/bigquery/README.md

```diff
@@ -60,17 +60,36 @@ source:
+| `resource.urn`                 | `project_id.dataset_name.table_name`                                                                                                                                                                                                                |
+| `resource.name`                | `table_name`                                                                                                                                                                                                                                        |
+| `resource.service`             | `bigquery`                                                                                                                                                                                                                                          |
+| `description`                  | `table description`                                                                                                                                                                                                                                 |
+| `profile.total_rows`           | `2100`                                                                                                                                                                                                                                              |
+| `profile.usage_count`          | `15`                                                                                                                                                                                                                                                |
+| `profile.joins`                | [][Join](#Join)                                                                                                                                                                                                                                     |
+| `profile.filters`              | [`"WHERE t.param_3 = 'the_param' AND t.column_1 = \"xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx\""`,`"WHERE event_timestamp >= TIMESTAMP(\"2021-10-29\", \"UTC\") AND event_timestamp < TIMESTAMP(\"2021-11-22T02:01:06Z\")"`]                                  |
+| `schema`                       | [][Column](#column)                                                                                                                                                                                                                                 |
+| `properties.partition_field`   | `created_at` - returns the time partitioning field md.TimePartitioning.Field                                                                                                                                                                        |
```

Comment:
I don't think it would be clear as to what `md.TimePartitioning.Field` means here.

Reply:
Done